pkg/config/default.go

```diff
@@ -286,6 +286,7 @@ func defaultConfigFromMemory() (*EngineConfig, error) {
 	c.CgroupManager = defaultCgroupManager()
 	c.ServiceTimeout = uint(5)
 	c.StopTimeout = uint(10)
+	c.ExitCommandDelay = uint(5 * 60)
```

Comment:
Minor nit, feel free to ignore: A comment that this is 5 minutes (5 * 60 seconds) would be nice

Reply:
I'm going to ignore this for now as it is already documented three times and no other defaults have any comments. If other people want a comment here, let me know and I'll be happy to put it there